rdbms/src/main/java/org/spine3/server/storage/jdbc/aggregate/query/AggregateStorageQueryFactory.java

```diff
@@ -32,11 +32,10 @@
 import static org.spine3.server.storage.jdbc.aggregate.query.Constants.EVENT_COUNT_TABLE_NAME_SUFFIX;
 
 /**
- * This class provides public methods for creating the most commonly used queries.
- * To use you custom queries extend this class and overwrite necessary methods.
- * See {@link JdbcStorageFactory} to see how to pass your custom factory to the storage
+ * This class creates the most commonly used queries.
```

Comment:
`Creates queries for aggregate storage.`

Why only common queries? Are there any queries which are created without this factory? I don't think so.